src/components/ScholarshipAmountField.jsx

```diff
@@ -35,18 +43,22 @@ function ScholarshipAmountField(props) {
           <div className={classes.amountFieldStyle}>
             Range:
             <AmountTextField
-              value={minAmount || ''} // hack so that the placeholer 'Unknown' shows up instead of the default value of 0
-              onChange={(e) =>
-                updateAmount(parseInt(e.target.value, 10) || '', maxAmount)
-              }
+              error={amountType === AmountType.Range && minAmountError}
+              value={minAmount || ''}
+              onChange={(e) => {
+                const val = parseInt(e.target.value, 10);
+                updateAmount(val % 1 === 0 ? val : 0, maxAmount);
```

Comment:
`val % 1` always returns `0` so this can be written:
```suggestion
                updateAmount(val, maxAmount);
```
If you're checking for `NaN`:
```suggestion
                updateAmount(val === NaN ? val : 0, maxAmount);
```
Or more succinctly I think you can do:
```suggestion
                updateAmount(val || 0, maxAmount);
```

Reply:
Basically what you had was fine, just needed to change `''` to `0`

Reply:
I initially had this to make sure the inputted value wasn't a decimal but the new implementation doesn't allow characters such as `.` so the first option works.